src/platform-aws.cpp

```diff
@@ -161,6 +139,29 @@ static struct ec2_platform_data platform_data_map[] = {
 			{ "NCCL_NET_FORCE_FLUSH", "0" },
 		},
 	},
+	{
+		.name = "p5en-and-later",
```

Comment:
Not a huge fan of this name change.  We print this, and it won't always be right.  Just leave the generic catch all and don't over-think it.